digital_asset_types/src/dao/generated/sea_orm_active_enums.rs

```diff
@@ -163,3 +163,16 @@ pub enum Instruction {
     #[sea_orm(string_value = "verify_creator")]
     VerifyCreator,
 }
+
+#[derive(Debug, Clone, PartialEq, EnumIter, DeriveActiveEnum, Serialize, Deserialize)]
+#[sea_orm(
+    rs_type = "String",
+    db_type = "Enum",
+    enum_name = "last_requested_status_code"
+)]
+pub enum LastRequestedStatusCode {
+    #[sea_orm(string_value = "success")]
+    Success,
+    #[sea_orm(string_value = "failure")]
+    Failure,
+}
```

Comment:
I was thinking the different http status codes like 403 500, 200. Just saving the status code number instead will suffice.